### DISCUSSION TITLE
Any end-user getting-started guide?

### DISCUSSION THREAD

**Discussion by romanrm** [freenet/freenet-core#896] [Q&A] [answered]
In documentation I see a discussion of HTTPS, WebSockets, components, contracts, API description.
Any getting-started guide for an end-user? How to run Freenet and what cool stuff can we try out?
Neither the "User Manual", nor the GitHub start page have a tiniest bit of info about actually setting up and using the application.
Freenet seems to be a familiar name, it is hard to believe there is only a kind of SDK and nothing usable to try right now.

**Comment by iduartgomez** [comment, marked as answer]
Hi there, the old classic freenet (renamed to hyphanet, https://www.hyphanet.org/) has an operation network, this version currently still has not a working network (we are workign right now on rolling the first test network), so there is no end user installable still. We hope we will get to that point early 2024.

Hope that answers your question.
